network/vpc/reference-content/use-case-basic.mdx

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  title: VPC use case 1 - Basic infrastructure to leverage VPC isolation
+  description: Learn how to set up a basic infrastructure using VPC isolation for secure cloud environments. Step-by-step guidance on leveraging VPCs for optimal network isolation.
+content:
+  h1: VPC use case 1 - Basic infrastructure to leverage VPC isolation
+  paragraph: Learn how to set up a basic infrastructure using VPC isolation for secure cloud environments. Step-by-step guidance on leveraging VPCs for optimal network isolation.
+tags: vpc private-network connectivity best-practice use-case infrastructure-diagram
+dates:
+  validation: 2024-10-25
+  posted: 2024-10-25
+categories:
+  - network
+---
+
+Scaleway **V**irtual **P**rivate **N**etwork (VPC) and the accompanying range of network products, are important building blocks within the Scaleway ecosystem. By creating your Scaleway resources within a carefully thought out VPC-based infrastructure, you can maximize security, resilience and efficiency. 
+
+Knowing how to create, combine and configure resources such as Private Networks with definable subnets, VPC routing with custom routes, IPAM to define and manage your IP addresses, and resources such as Load Balancers and Public Gateways to facilitate connectivity and traffic distribution, is invaluable in achieving control over your networking environment.
+
+This is the first in a series of use cases designed to show you potentially interesting and useful infrastructures, illustrating how to use VPC and other network products to achieve a robust, secure and scalable cloud environment.
+
+## Overview
+
+The architecture diagram below shows the infrastructure for this use-case.
+
+<Lightbox src="scaleway-vpc-infra-1.webp" size="large" alt="An architecture diagram shows how a Load Balancer inside a Scaleway VPC is attached to a Private Network. Also attached to the Private Network are three Instances (connected to Block Storage), a Managed Database, and a Public Gateway. Users access resources via a domain name which a Domains and DNS entry points to the Load Balancer's public flexible IP address. The Load Balancer distributes requests to the three Instances, which do not have public IP addresses. Instances can privately retrieve and store data in the Managed Database and Block Storage. The Public Gateway also has a flexible IP address, which allows admins to connect to resources in the Private Network via SSH bastion. The gateway's public flexible IP also allows connections to Scaleway resources outside the VPC, e.g. Object Storage, Transactional Email, Nats, Queues, Topics and Events, Serverless products, and Secret Manager." />
```

Comment:
"Users access resources via a domain name which a Domains and DNS entry points to the Load Balancer's public flexible IP address." I feel like something is missing between "which" and "a Domains and DNS"